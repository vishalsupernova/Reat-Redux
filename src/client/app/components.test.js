import  Operation  from './increment.js';
import React from 'react';
import renderer from 'react-test-renderer';
import { createStore } from 'redux';
import counter from './inreducer.js';
import { Provider } from 'react-redux';

describe('COMPONENT', ()=> {
    describe(<Operation />, () => {
        it('Render', () => {
            var tree = renderer.create(<Provider store = {createStore(counter)}><Operation/></Provider>).toJSON();
            expect(tree).toMatchSnapshot();
        })
    })
})