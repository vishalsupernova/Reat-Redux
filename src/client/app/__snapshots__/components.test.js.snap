// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT [object Object] Render 1`] = `
<div>
  <h3>
    0
  </h3>
  <button
    onClick={[Function]}
  >
    INC
  </button>
  <button
    onClick={[Function]}
  >
    DEC
  </button>
  <input
    onKeyUp={[Function]}
    placeholder="Increment"
    type="text"
  />
  <input
    onKeyUp={[Function]}
    placeholder="Decrement"
    type="text"
  />
</div>
`;
